boards/arm/samv7/common/src/sam_boot_image.c

```diff
@@ -59,9 +59,7 @@ struct arm_vector_table
  ****************************************************************************/
 
 static void cleanup_arm_nvic(void);
-#ifdef CONFIG_ARMV7M_SYSTICK
```

Comment:
Is systick mandatory? I mean, what if someone decide to use other timer instead systick?

Reply:
The NVIC systick is initialized in `up_timer_initialize()` no matter if `CONFIG_ARMV7M_SYSTICK`, so we should disable it in BL anyway.
Regarding the tickles mode support, I will try to cover this case in the next PR.

Reply:
Ok, gotcha!